doc/design.rst

```diff
@@ -606,3 +597,61 @@ are deleted, the particular snapshot vanished and all snapshots
 depending on data that has been added in the snapshot cannot be restored
 completely. Restic is not designed to detect this attack.
 
+With the aforementioned assumptions and guarantees in mind, the following  are
+examples of things an adversary could achieve in various circumstances.
+
+An adversary with access to your backup storage location could: 
+- Delete or manipulate your backups, thereby impairing your ability to restore
+files from the compromised storage location. - Attempt a brute force password 
+guessing attack against a copy of the  repository (even more reason to use 
+long, random passwords).
+
+An adversary with write access to your files at the storage location could: 
+- Delete specific files (by using timestmaps to determine which files belong to
+what snapshots), resulting in snapshots vanishing and all dependent snapshots
```

Comment:
"preventing" is missing between all and dependent.

Reply:
Good catch, but I'm thinking "preventing: should be between "and" and "all" instead (i.e., resulting in snapshots vanishing and preventing all dependent snapshots from being completely restored).